config/config-mp/src/main/java/io/helidon/config/mp/MpConfigBuilder.java

```diff
@@ -492,7 +498,12 @@ public Config build() {
 
         // nope, return the result
         if (configuredProfile == null) {
+            LOGGER.fine("Built MP config with no profile");
```

Comment:
`if (LOGGER.isLoggable(Level.FINEST)) {`

Reply:
Not needed (also it is fine not finest). If the string is a constant, there is not much overhead doing it this way. The guarding of logs statements should only be used when passing variables to output